libcloud/compute/drivers/openstack.py

```diff
@@ -1588,7 +1588,7 @@ def ex_resize(self, node, size):
 
         :rtype: ``bool``
         """
-        server_params = self._create_args_to_params(node, size=size)
+        server_params = {'flavorRef': size.id}
```

Comment:
P.S. Next time it would help if you can link to the corresponding API documentation (e.g. https://developer.openstack.org/api-ref/compute/?expanded=resize-server-resize-action-detail in this case).